### PR TITLE
add parameter 'connectionPoolTimeout' in both SmartConnect and SmartConnectNoSSL

### PR DESCRIPTION
**Pull request by magianxd**
add parameter 'connectionPoolTimeout' in both SmartConnect and SmartConnectNoSSL

This PR is for issue #347, @tianhao64

### CODE REVIEW

File: pyVim/connect.py [hunk 1]
```diff
@@ -192,6 +192,7 @@ def _doLogin(soapStub):
 
 def Connect(host='localhost', port=443, user='root', pwd='',
             service="hostd", adapter="SOAP", namespace=None, path="/sdk",
+            connectionPoolTimeout=CONNECTION_POOL_IDLE_TIMEOUT_SEC,
```
Comment (tianhao64) [inline pyVim/connect.py]: Looks like connectionPoolTimeout was not passed to __Login and __LoginBySSPI

File: pyVim/connect.py [hunk 2]
```diff
@@ -391,7 +399,8 @@ def __Login(host, port, user, pwd, service, adapter, version, path,
 ## Copyright (c) 2015 Morgan Stanley.  All rights reserved.
 
 def __LoginBySSPI(host, port, service, adapter, version, path,
-                  keyFile, certFile, thumbprint, sslContext, b64token):
+                  keyFile, certFile, thumbprint, sslContext, b64token,
+                  connectionPoolTimeout=CONNECTION_POOL_IDLE_TIMEOUT_SEC):
```
Comment (tianhao64) [inline pyVim/connect.py]: this is not passed to __RetrieveContent

File: pyVim/connect.py [hunk 3]
```diff
@@ -334,7 +338,8 @@ def GetLocalTicket(si, user):
 ## connected service instance object.
 
 def __Login(host, port, user, pwd, service, adapter, version, path,
-            keyFile, certFile, thumbprint, sslContext):
+            keyFile, certFile, thumbprint, sslContext,
+            connectionPoolTimeout=CONNECTION_POOL_IDLE_TIMEOUT_SEC):
```
Comment (tianhao64) [inline pyVim/connect.py]: this was not passed to __RetrieveContent

**Comment by tianhao64** [comment]
@magianxd Thanks a lot for the change!

**Comment by magianxd** [comment]
@tianhao64 When this change will be included in pypi package?

**Comment by tianhao64** [comment]
This will be part of next bug fix release. I don't have a specific date at this point. Will discuss with the team and update here.

**Pull request merged**